include/libswiftnav/ephemeris.h

```diff
@@ -25,18 +26,41 @@ typedef struct {
   gps_time_t toe, toc;
   u8 valid;
   u8 healthy;
-  u8 prn;
+  signal_t sid;
   u8 iode;
+} ephemeris_kepler_t;
+
+typedef struct {
+  gps_time_t toe;
+  double pos[3];
+  double rate[3];
+  double acc[3];
+  u8 valid;
+  u8 healthy;
+  signal_t sid;
+  u8 iod;
+  u16 toa;
+  u8 ura;
+  double a_gf0;
+  double a_gf1;
+} ephemeris_xyz_t;
+
```

Comment:
I also thought we were planning to have an enum and a union of pointers for this.